cpp/util/protobuf_util.cc

```diff
@@ -0,0 +1,51 @@
+#include "util/protobuf_util.h"
+
+#include <google/protobuf/io/coded_stream.h>
+#include <google/protobuf/io/zero_copy_stream_impl.h>
+#include <google/protobuf/message.h>
+
+namespace cert_trans {
+
+using google::protobuf::MessageLite;
+using google::protobuf::io::CodedOutputStream;
+using google::protobuf::io::OstreamOutputStream;
+using google::protobuf::io::ZeroCopyOutputStream;
+
+
+// Implements the MessageLite.writeDelimitedTo() method from the Java proto
+// API, which is strangly absent from the C++ library.
```

Comment:
It's got this mix of "strangling" in there, which is probably not entirely inaccurate either... :wink: